src/app/store/api.js

```diff
@@ -3,10 +3,12 @@ import { createClient } from 'contentful-management'
 let client = null
 let space = null
 
-export async function initClient (accessToken, spaceId) {
+export async function initClient (accessToken, spaceId, host, uploadHost) {
   try {
     client = createClient({
-      accessToken
+      accessToken,
+      defaultHostname: host,
+      defaultHostnameUpload: uploadHost
```

Comment:
But thats a undocumented config. Officiall users are only able set the host:
https://contentful.github.io/contentful-management.js/contentful-management/3.0.2/typedef/index.html#static-typedef-ContentfulManagement

And the host can be with port or without, both variants work :)